### DISCUSSION TITLE
Golang update 1.15.6

### DISCUSSION THREAD

**Discussion by ghost** [Homebrew/discussions#228] [Everyday usage]
Does Homebrew have to latest upgrade for Go yet?. I have 1.15.5 but when I try to upgrade via homebrew, it doesn't seem to be able to find the lastest version

**Comment by carlocab** [comment]
There's an open PR for it: https://github.com/Homebrew/homebrew-core/pull/66219

It's currently failing some tests, which is why it hasn't been merged. Maybe you can help out.

More generally, if you're wondering whether Homebrew has the newest version of anything, the pull requests would be the first place to look. If there isn't one, it would be helpful if you open one.

**Comment by abeltay** [comment]
There's an open PR at https://github.com/Homebrew/homebrew-core/pull/66355
